apps/desktop/src/app.css

```
@@ -3421,3 +3421,289 @@ body {
+}
+.rev-panel__count {
+  margin-left: auto; flex: 0 0 auto;
+  font-family: var(--font-mono);
+  font-size: 10px; letter-spacing: 0.05em;
+  color: var(--text-3);
+}
+.rev-panel__count b { color: var(--text-2); font-weight: 600; }
+.rev-panel__close { flex: 0 0 auto; }
+.rev-panel__close kbd {
+  font-family: var(--font-mono); font-size: 9px; font-weight: 600;
+  border: 1px solid currentColor; opacity: 0.5;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)